src/test/kotlin/no/nav/meldeplikt/meldekortservice/api/PersonKtTest.kt

```diff
@@ -307,15 +302,11 @@ class PersonKtTest {
         meldekortKontrollertType.status = "OK"
         meldekortKontrollertType.arsakskoder = MeldekortKontrollertType.Arsakskoder()
 
-
         mockkObject(SoapConfig)
-        val externControlEmeldingSOAP = mockk<ExternControlEmeldingSOAP>()
-
-        every { SoapConfig.soapService() } returns SoapServiceImpl(externControlEmeldingSOAP, mockk())
-        every { externControlEmeldingSOAP.kontrollerEmeldingMeldekort(any()) } throws RuntimeException("Error i arena")
+        every { SoapConfig.soapService() } returns SoapServiceImpl(mockk())
```

Comment:
Det er vel ikke noe Soap som er aktuelt for denne testen?

Reply:
Det stemmer. Slettet.